api/python/provisioner/__main__.py

```diff
@@ -39,6 +39,9 @@
     log
 )
 from .commands import commands
+from .commands.setup_provisioner import (
+    SetupCmdBase, RunArgsSetupProvisionerGeneric
```

Comment:
@andkononykhin 
This was the reason the commit was reverted.
Due to this, nodes was expected as a mandatory positional argument and the commands (none of the API ones) never proceeded beyond this point.

We need to understand the flow and fix it